README.md

```diff
@@ -142,12 +142,12 @@ All that is asked is that you have at least one previous contribution to RES.
 
 ##### Building in Safari (assumes Mac)
 
-  1. Open the `Preferences` by going to `Safari->Preferences` or pressing `⌘`, then go to `Advanced` and check the checkbox for `Show develop menu in menu bar`.
-  2. Navigate to `Develop->Show Extension Builder` to open the extensions menu. Add a new extension by pressing the `+` in the bottom left and choosing `Add extension`.
+  1. Open the `Preferences` by going to `Safari->Preferences` or pressing `⌘,` then go to `Advanced` and check the checkbox for `Show Develop menu in menu bar`.
```

Comment:
The comma should be outside the \`, not inside it (unless I misunderstand my grammar rules).